browser/main/lib/dataApi/attachmentManagement.js

```diff
@@ -584,6 +584,7 @@ function deleteAttachmentFolder (storageKey, noteKey) {
  * @param noteKey NoteKey of the current note. Is used to determine the belonging attachment folder.
  */
 function deleteAttachmentsNotPresentInNote (markdownContent, storageKey, noteKey) {
+  console.log('deleteAtt')
```

Comment:
This one!

Reply:
ups.. stupid me... i thought i had removed that..

Reply:
done